### PR TITLE
Readme update

### PR DESCRIPTION
**Pull request by ric-e**
@jackkoskie silly excuse for practicing pull requests but also putting changes on the README to reflect our concept plan

### CODE REVIEW

**Comment by ric-e** [comment]
two commits because im dum shh

File: README.md [hunk 1]
```diff
@@ -1,18 +1,55 @@
...
+  * Manual input of simple information on pill bottles
+  * Uses data to create calendar reminders 
+  * Adjustable whenever, however you want
+
+## Future Features
+
+Although this web application would aim to be succesful to its intended core audience, a mobile applicaiton that is compatible to any device would be a future goal. Furthermore, this web app would benefit from Optical Character Recognition (OCR) input, facilitating how data woudl be collected from pill bottles and overcoming language barriers, especially in the elderly audience. This would require a complex algorithim to learn how to read the info, as not all bottle labes are in the same layout or on a planar surface. 
+
+
+## Development History
```
Comment (jackkoskie) [inline README.md]: Development History section is not required - git tracks this for us

File: README.md [hunk 2]
```diff
@@ -1,18 +1,55 @@
-# sv
+# Chill Pill
+
+### A medication reminder and calender web application
```
Comment (jackkoskie) [inline README.md]: line 3: this shouldn't be a heading, remove ###

**Review by jackkoskie — CHANGES_REQUESTED** [review]
See comments, looks good overall! 

I think this method of reviewing each others code will work well. I'll create some "rules" in Github to kinda enforce this practice however it will always be able to be overridden by any of us

**Comment by ric-e** [comment]
All suggested changes made, let me know if this is good

**Comment by jackkoskie** [comment]
Looks good @ric-e! I'll merge it

**Review by jackkoskie — APPROVED** [review]

**Pull request merged**